src/stores/messages/MessagesUpdateStore.ts

```diff
@@ -50,10 +50,10 @@ export default class MessagesUpdateStore {
 				resultCountLimit: undefined,
 			},
 			{
-				onResponse: incommingMessages => {
-					if (incommingMessages.length) {
-						onNextChannelResponse(incommingMessages);
-						this.scrollToMessage(incommingMessages[0].messageId);
+				onResponse: incomingMessages => {
+					if (incomingMessages.length) {
+						onNextChannelResponse(incomingMessages);
+						this.selectedMessageId = incomingMessages[0].messageId;
```

Comment:
change that to messageStore.selectedMessageId and remove `private selectedMessageId: String | null,`